client/src/pages/TaggingPage.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { useEffect } from "react";
 import styled from "styled-components/macro";
+import PropTypes from "prop-types";
 import { Button } from "../components/Button";
 import { addNewTag } from "../utils/api";
 import { getImageObj } from "../utils/api";
```

Comment:
```suggestion
import { addNewTag , getImageObj } from "../utils/api";
```

just to have fewer lines, you can do the same here ;)